apps/designer/app/canvas/shared/use-drag-drop.ts

```diff
@@ -8,7 +8,7 @@ import {
   usePlacement,
   useDrag,
   useDropTarget,
-} from "@webstudio-is/design-system";
+} from "@webstudio-is/design-system/src/components/primitives/dnd";
```

Comment:
needs to be exported from index in design-system, can't go inside any folder that has index, we need a linter for this

Reply:
Thought it would be easier for naming if DnD wouldn't share namespace with everything else in design-system. Maybe we should do something like this? 

```js
import { dnd } from "@webstudio-is/design-system";
const { useDrag } = dnd;
```

This will likely break tree-shaking though.

Reply:
I don't think we will have a namespaces problem even if we export useDrag straight, because there won't be another drag&drop solution there, but if there will be, we can always do what you just said, not sure regarding tree shaking there, I mean if you import dnd you will likely use all its hooks, so ...